### DISCUSSION TITLE
Prevent other apps accessing my backend

### DISCUSSION THREAD

**Discussion by FNG-2002** [pocketbase/pocketbase#2706] [Q&A] [answered]
Hello everyone, is there a way to prevent someone from creating an app that accesses my Pocketbase backend and pretends to be the original app? For example, Firebase offers the feature AppCheck. Each app has a certificate that is verified by the server before any queries can be made. Has something like this already been implemented or is it planned? Thank you.

**Comment by ganigeorgiev** [comment, marked as answer]
There is no builtin functionality like AppCheck at the moment and at least for the near future I don't think I'll have the time to work on it (for the currently planned tasks you could check the [public roadmap](https://github.com/orgs/pocketbase/projects/2/views/1)).

Note that it doesn't necessary need to be builtin in PocketBase and you can implement your own attestation mechanism (eg. based on time-based tokens) when using as framework or even on a reverse proxy level (_although I imagine it would be a little more involving_).
